pip/download.py

```diff
@@ -652,7 +615,11 @@ def _copy_file(filename, location, content_type, link):
         logger.info('Saved %s', display_path(download_location))
 
 
-def unpack_http_url(link, location, download_dir=None, session=None):
+def unpack_http_url(link,
```

Comment:
What's the preferred style? I did what we usually do at Mozilla, but I don't really care. Does pypa like to cram as many args onto each line as possible?